docs/api-markdown-documenter/render-api-documentation.js

```diff
@@ -29,7 +29,12 @@ async function renderApiDocumentation(version) {
 		version,
 		"_build",
 	);
-	const apiDocsDirectoryPath = path.resolve(__dirname, "..", "content", "docs", "apis", version);
+
+	// TODO: remove check for 2.0 and just set apiDocsDirectoryPath to include version.
```

Comment:
I think this will be a problem if we try to run `build:multi-version`, since we'll end up with a mix of `docs/apis/*` and `docs/apis/1.0/*`.

I wonder if we need this function to take in an `inputDir` and an `outputDir` from the caller (in addition to the version number). Then we could have `index.js` set up the right directories based on whether or not we're doing a multi-version build or a single-version build. Would that make sense?